packages/core-app-api/src/routing/RouteTracker.tsx

```diff
@@ -56,13 +57,23 @@ const getExtensionContext = (
       routeRef = routeObject.routeRefs.values().next().value;
     }
 
+    const params = Object.entries(
+      routeMatch?.params || {},
+    ).reduce<AnalyticsEventAttributes>((acc, [key, value]) => {
+      if (value !== undefined) {
+        acc[key] = value;
```

Comment:
Would be great to test that these are being captured as attributes on the `navigate` event.  There should already be tests in place somewhere that track the `navigate` event being fired at all!  An extra case or two there would be good.